tests/core.test.ts

```diff
@@ -17,10 +17,35 @@ describe('Safe Core SDK', () => {
     }
   })
 
+  describe('connect', async () => {
```

Comment:
nitpick: 
in tests for connect you describe tests as `it('connect...)` whether in other tests you do `it('should do something')`, would be great to stick to something (I like 2nd more personally)